net/airos-dfs-reset/files/airos-dfs-reset

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+. /lib/functions.sh
+
+log() {
+    local msg="$1"
+    logger -t airos-dfs-reset -s "$msg"
```

Comment:
"-s" is going to stderr. Do you really want to log everything to stderr? ;)

Reply:
Indeed, for manual execution. If executed via procd it should be ignored, isnt it?